### PR TITLE
Removal of widgets that did not contribute to the UI 

### LINKED ISSUE #718: Reduce unnecessary wrapping of child widgets.
- Parent issue : #715 
- **Issue Description:** <Add brief description about issue or feature you want to add.>
Reducing unnecessary wrapping of child widgets in all the files when not needed and when the desired output could be achieved with the child property itself eg: Container wrapped in Align or Padding widget, Nested Row, and Columns``` etc.
- **Issue Severity:** <High or Medium orLow>
Low
- **Environment Details:** <Add details about environment where this issue is found.>
Project Level
- **Expected Behaviour:** <Add what would like to see once this is issue is fixed.>
Removal of unused widgets, unused files. Generation of reusable widgets
- **Does this issue need immediate attention?** <If yes. Please explain in detail>
No
- **Please ask mentors/maintainers to assign if you are willing to work on this issue.**

### PR DESCRIPTION
**Pull request by nayakastha**
**What kind of change does this PR introduce?** This PR removed all the unnecessary widgets and refactored the toasts to individual widget files since it is used multiple times in the app. 

**Did you add tests for your changes?** No, It was not required. Though I debugged it using dart dev tools and adb device.

**If relevant, did you update the documentation?** Not Applicable

**Summary** -  Fixes #718 

**Does this PR introduce a breaking change?** No it is just minor refactoring

**Other information**

### CODE REVIEW

**Comment by sumitra19jha** [comment]
Hi @nayakastha 
In `analysis_options.yaml`, currently we have set `avoid_unnecessary_containers: false`
As you are working on this issue, I would recommend you to set it to `true` and it will help you finding out all unnecessary container. Run `flutter analyze` after setting it too true.

**Comment by nayakastha** [comment]
> Hi @nayakastha
> In `analysis_options.yaml`, currently we have set `avoid_unnecessary_containers: false`
> As you are working on this issue, I would recommend you to set it to `true` and it will help you finding out all unnecessary container. Run `flutter analyze` after setting it too true.

Thank you @sumitra19jha. I will do the needful.

File: lib/controllers/post_controller.dart [hunk 1]
```diff
@@ -23,22 +23,22 @@ class PostController with ChangeNotifier {
...
     final String currentOrgID = await preferences.getCurrentOrgId();
     final String currentUserID = await preferences.getUserId();
     this.currentUserID = currentUserID;
     final String query = Queries().getPostsById(currentOrgID);
     final Map result = await apiFunctions.gqlquery(query);
-    print(DateTime.now().difference(d1));
+
     if (result != null) {
-      print(posts.isEmpty);
+      debugPrint(posts.isEmpty.toString());
```
Comment (sumitra19jha) [inline lib/controllers/post_controller.dart:40]: This is a much needed change. It is not advisable to have print statement in production stage of the app.

Comment (nayakastha) [reply lib/controllers/post_controller.dart:40]: I assume there is an issue regarding this. I hope the changes I have made solves them.

**Comment by Sagar2366** [comment]
@nayakastha thank you for working on this issue.
Please resolve conflicts on this PR and remove commented out code which is not required.

**Comment by Sagar2366** [comment]
@nayakastha few suggestions :
1. Do not add debugPrint statements in same PR, it is different feature and should be handled separately.
2. Breakdown the PR for separate files, create child issues and raise separate small PR's. I t would be more easier to review small PR's.
3. Please resolve conflicts on this PR.

**Comment by nayakastha** [comment]
> @nayakastha few suggestions :
> 
> 1. Do not add debugPrint statements in same PR, it is different feature and should be handled separately.
> 2. Breakdown the PR for separate files, create child issues and raise separate small PR's. I t would be more easier to review small PR's.
> 3. Please resolve conflicts on this PR.

The debugPrint issue was made after I made changes into this, So I missed out, I am really sorry for the chaos. I cleared all the conflicts should be good to go now.

**Comment by nayakastha** [comment]
@Sagar2366 sir, I hope all the issues in this PR and clear and good to go for a merge.

**Comment by palisadoes** [comment]
@nayakastha please review the conflicts.

**Comment by nayakastha** [comment]
> @nayakastha please review the conflicts.

Sir, I don't seem to have the write access for this branch now. It says I don't have write access

**Comment by nayakastha** [comment]
@palisadoes sir solved the conflicts of this PR as well

**Pull request merged**